### PR TITLE
Fixing insert of unneeded html tags in db

### PR DESCRIPTION
**Pull request by akuma06**
Added a Sanitize function in util
* Possibility to add model in it
* Already a preset default model

Comments shouldn't be allowed html, too difficult to check every comment
for broken html
Torrents are still allowed html tags but I don't think it should since
we use markdown.

### CODE REVIEW

**Comment by yiiTT** [comment]
This is a tricky issue, I think we need to look at it a bit more, but not allowing HTML for comments.
But for the Description I don't think you should use the Strict Policy since this limits their tools the UGCPolicy should be more than enough.

**Comment by yiiTT** [comment]
Suggested changes: re-implement the UGCPolicy without div tags for upload.go and then apply the custom policy on view_torrent_handler.go so we don't mess with what goes into the DB too much.

File: util/markdown.go [hunk 1]
```diff
@@ -38,3 +38,24 @@ func MarkdownToHTML(markdown string) template.HTML {
...
+ * Sanitize a message passed as a string according to a setted model or allowing a set of html tags and output a string
+ */
+func Sanitize(msg string, elements ...string) string {
+	p := bluemonday.StrictPolicy()
+	if len(elements) > 0 {
+		if elements[0] == "default" { // default model
+			p.AllowElements("b", "strong", "em", "i", "u", "blockquote", "q")
+			p.AllowImages()
+			p.AllowStandardURLs()
+			p.AllowAttrs("cite").OnElements("blockquote", "q")
```
Comment (sfan5) [inline util/markdown.go]: do people ever use this?

Comment (yiiTT) [inline util/markdown.go]: yea other than cite

**Pull request merged**